projects/packages/connection/src/class-error-handler.php

```diff
@@ -199,15 +199,16 @@ public static function get_instance() {
 	 *
 	 * @since 1.14.2
 	 *
-	 * @param \WP_Error $error the error object.
-	 * @param boolean   $force Force the report, even if should_report_error is false.
+	 * @param \WP_Error $error  The error object.
+	 * @param boolean   $force  Force the report, even if should_report_error is false.
+	 * @param boolean   $verify_locally Set to 'true' to verify the error locally and skip the WP.com verification.
 	 * @return void
 	 */
-	public function report_error( \WP_Error $error, $force = false ) {
+	public function report_error( \WP_Error $error, $force = false, $verify_locally = false ) {
```

Comment:
nitpick suggestion. I think `$skip_wpcom_verification` would be a better name for this

Reply:
Good point, renamed: https://github.com/Automattic/jetpack/pull/25694/commits/1153aad07a019971c315ae969b058bdbc055e99d